graph/articles/collections.md

```diff
@@ -316,3 +316,151 @@ Content-Type: application/json
    "value": []
 }
 ```
+
+## 11. Collections of structural types (complex types or primitive types)
+
+Collections of entity types are generally preferable to collections of structual types because collections of structural types must be updated as a single unit, meaning that they are overwritten entirely by new contents, rather than be updated relative to the existing contents. 
```

Comment:
this is one of the reasons entities are favorable over complex types within a collection, but limiting to just this case often gets us in trouble.  i.e., a workload may say "this is read-only", or "I'm okay with them replacing the whole collection", and then get into trouble later, either because they do need to update an individual member (or something nested within an individual member), or because they need to refer to an individual member for some other reason (i.e., an auditLog or recommendation might want to reference an individual key credential). Also, it's harder to page collections of complex types.

So, for a variety of reasons, the strong recommendation is to use entity types in collections.